ivo-projects/todo/src/reducers/index.js

```diff
@@ -26,6 +26,16 @@ const editByID = (reminderList = [], id, payload) => {
     return updatedReminderList;
 };
 
+const editReminderTextById = (reminderList = [], id, text) => {
```

Comment:
Use the editByID function (rename it to updateTodoById)

Reply:
@RaulTsc yes...but I have two separated action for each function..that would imply reduce the whole feature logic to one single action

Reply:
It's ok to have two actions, but the reducer case should used the above mentioned function